### DISCUSSION TITLE
Action counter for producing items that go by X actions

### DISCUSSION THREAD

**Discussion by IromTypo** [runelite/runelite#15787] [Q&A] [answered]
So when you make broad arrows, you have to either count the exp drops or wait until the loop is finished. Either you have to be completely focused on the game (for forever as an ironman) or waste exp.

Now the solution is simple: a visual indicator, a counter! it would be really cool if someone could make that for things you produce in a stack! Of course Javelin Shafts with Javelin Heads and Arrow Shafts with arrowheads are 10 actions, other than that I don't really know if there are more use cases in the game. Would like some feedback, but even if no other use, still a plugin I would a lót!

Thank you :)

**Comment by Enriath** [comment, marked as answer]
A plugin like this exists on the hub already: [Action Progress](https://runelite.net/plugin-hub/show/action-progress)

**Reply by IromTypo** [reply]
Thank you!!
